specification/quantum/data-plane/Microsoft.Quantum/preview/2019-11-04-preview/quantum.json

```diff
@@ -0,0 +1,552 @@
+        ],
+        "x-ms-examples": {
+          "Get list of of providers and their status for a Quantum Workspace": {
+            "$ref": "./examples/providerStatus.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "The operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/providerStatusList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/restError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/v1.0/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Quantum/workspaces/{workspaceName}/storage/sasUri": {
+      "post": {
+        "operationId": "Storage_SasUri",
+        "description": "Gets a URL with SAS token for a container/blob in the storage account associated with the workspace. The SAS URL can be used to upload job input and/or download job output.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/workspaceNameParameter"
+          },
+          {
+            "name": "blobDetails",
+            "description": "The details (name and container) of the blob to store or download data.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/blobDetails"
+            }
+          }
+        ],
+        "x-ms-examples": {
+          "Get a SAS token to upload data for a Quantum Workspace": {
+            "$ref": "./examples/storageGet.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/sasUriResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/restError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "jobDetailsList": {
+      "description": "List of job details.",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/jobDetails"
+          }
+        },
+        "count": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Total records count number."
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Link to the next page of results."
+        }
+      }
+    },
+    "jobDetails": {
```

Comment:
I would strongly suggest that you call this model "Job". It is what you get if you do a GET on the job path after all. 
While this is not impacting the REST API, but it would affect generated models and documentation etc.